app/config.py

```diff
@@ -0,0 +1,17 @@
+"""Flask configuration."""
+from os import environ
+from os import path
+
+from dotenv import load_dotenv
+
+basedir = path.abspath(path.dirname(__file__))
```

Comment:
Would this piece of code lead to one storing the .env file in the /app directory? 

I think that typically configs are stored in the root, so that configuration and source code are separated. Although that is a bigger concern if one is building the project into a wheel or some other package file - since you don't want to include configuration in there.

What are the benefits of using a class to store configuration over a yaml/toml file?

Reply:
Yes agreed and good spot, either this file (and definitely the .env file if used) should be in the root. The benefits of a class are that you can have a base config and then inherit the base config in different classes which automatically get loaded in different environments (with perhaps just one prod/dev environment set to toggle between them. However having lots of different environment variables set in the application at the moment is overkill. So perhaps easiest for now is to remove this (also as it is unused) and just take the environment variables from the global environment or defaults if set